pandas/io/excel.py

```diff
@@ -902,7 +902,7 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0):
 
         for cell in cells:
             colletter = get_column_letter(startcol + cell.col + 1)
```

Comment:
all of these changes need to be back-compat. so need to switch code based on the version.